tools/project_manager/edit_common.inc

```diff
@@ -15,10 +15,13 @@ function just_echo( $field_value )
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
-function text_field( $field_value, $field_name )
+function text_field( $field_value, $field_name, $args=array())
 {
+    $maxlength = array_get($args, "maxlength", NULL);
```

Comment:
Since the default is NULL, you could just say
`$maxlength = @$args["maxlength"];`

Reply:
I've come to dislike using `@` for this purpose and would prefer we stop using it in most all cases. It's like blindly catching and ignoring exceptions in python. I see this as a bad programming practice and would rather explicitly handle cases rather than swallowing errors.

Reply:
Yeah, I can agree to that.

Add it to https://www.pgdp.net/wiki/DP_Code_Best_Practices#Coding_conventions ?

Reply:
Added as https://www.pgdp.net/wiki/DP_Code_Best_Practices#Squashing_errors.2Fwarnings adjust wording as you see fit.